### DISCUSSION TITLE
Self Hosted Solution

### DISCUSSION THREAD

**Discussion by hirbod** [flyerhq/react-native-chat-ui#42] [Ideas / Feature Requests] [answered]
A self hosted Socket solution as example would be great. Ideally using Socket.io or SocketCluster.
I would skip the part of storage into a database, because the sky is the limit here (php, graph, sql, etc.)

**Comment by demchenkoalex** [comment, marked as answer]
Hey @Hirbod we wanted to create a service actually (paid one, self host and cloud) to support the open source initiatives. But things didn't work out for now and I only support the UI library, everything else is will be too much for me.
